### PR TITLE
Run Tests and Linter on Pull requests

### PR DESCRIPTION
**Pull request by tobiweisss**
chore: run pytest and flake8 on every pull request and on every push to main

### CODE REVIEW

File: LeBotTel/LeBotTel/base_telegram_bot.py
Comment (tobiweisss) [inline LeBotTel/LeBotTel/base_telegram_bot.py:1]: Just Formatting

File: LeBotTel/LeBotTel/base_message_handler.py
Comment (tobiweisss) [inline LeBotTel/LeBotTel/base_message_handler.py:1]: Just Formatting

File: LeBotTel/LeBotTel/exceptions.py
Comment (tobiweisss) [inline LeBotTel/LeBotTel/exceptions.py:1]: Just Formatting

File: tests/conftest.py
Comment (tobiweisss) [inline tests/conftest.py:1]: Just Formatting

File: tests/test_base_classes.py
Comment (tobiweisss) [inline tests/test_base_classes.py:1]: Just Formatting

File: tests/test_base_message_handler.py
Comment (tobiweisss) [inline tests/test_base_message_handler.py:1]: Just Formatting

**Review by SpaceGnu — APPROVED** [review]

**Pull request merged**